clients/roscpp/include/ros/node_handle.h

```diff
@@ -289,7 +289,9 @@ namespace ros
    * \return On success, a Publisher that, when it goes out of scope, will automatically release a reference
    * on this advertisement.  On failure, an empty Publisher which can be checked with:
 \verbatim
-if (handle)
+ros::NodeHandle nodeHandle;
+ros::publisher pub = nodeHandle.advertise<std_msgs::Empty>("my_topic", 1, (ros::SubscriberStatusCallback)callback);
+if (pub)	// Enter if publisher is valid
```

Comment:
Please replace the tab before the `//` with two spaces. Same for all the other blocks.